### PR TITLE
minor fixes, added local node docs, syntax highlighting (JavaScript + TypeScript)

### PR DESCRIPTION
**Pull request by dabit3**
- Added syntax highlighting for JS + TS
- Added local node documentation
- Fixed error in `test/harness.rs`

### CODE REVIEW

**Review by camiinthisthang — DISMISSED** [review]
LGTM, will wait for @sarahschwartz to approve before we merge

File: beta2-quickstart-master/counter-contract/tests/harness.rs
Comment (sarahschwartz) [inline beta2-quickstart-master/counter-contract/tests/harness.rs]: Underscores are used for unused variables in Rust, so I think we should keep this as-is.

**Review by sarahschwartz — CHANGES_REQUESTED** [review]
looks great, thank you! Just a note about the updates to the test harness

**Comment by dabit3** [comment]
Thanks @sarahschwartz @camiinthisthang should be gtg now :)

**Comment by sarahschwartz** [comment]
@dabit3 It looks like our markdown linter failed because of these rules: no multiple consecutive blank lines, and files should end with a single newline character.

**Comment by dabit3** [comment]
Thanks @sarahschwartz I have made updates based on the linting errors, please have a look and let me know if this passes.

**Review by sarahschwartz — APPROVED** [review]

**Pull request merged**
